src/wallet/scriptpubkeyman.cpp

```diff
@@ -624,10 +624,17 @@ std::optional<MigrationData> LegacyDataSPKM::MigrateToDescriptor()
     for (const auto& chain_pair : m_inactive_hd_chains) {
         chains.push_back(chain_pair.second);
     }
+
+    int wallet_version = 0;
+    auto x = m_storage.GetDatabase().MakeBatch();
+    x->Read(DBKeys::MINVERSION, wallet_version);
+
+    bool can_support_hd_split_feature = wallet_version >= FEATURE_HD_SPLIT;
+
```

Comment:
```suggestion
```
I had a chat with @achow101 about it yesterday, she commented to me that the "_hd chain split versioning is stored in CHDChain too_" so we can get rid of `FEATURE_HD_SPLIT` and the the entire `enum` there.

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/32977/commits/269c43bdb04768954e59119363afdfa061849ed0